test/e2e/app-dir/use-cache-hanging-inputs/use-cache-hanging-inputs.test.ts

```diff
@@ -256,11 +278,19 @@ describe('use-cache-hanging-inputs', () => {
         const title = await getRedboxTitle(browser)
         const description = await getRedboxDescription(browser)
 
-        expect({ count, title, description }).toEqual({
-          count: 1,
-          title: 'Unhandled Runtime Error',
-          description: '[ Cache ] Error: kaputt!',
-        })
+        if (isNewDevOverlay) {
+          expect({ count, title, description }).toEqual({
+            count: 1,
+            title: 'Unhandled Runtime Error\nCache',
```

Comment:
same here, the badge should be a new property, otherwise it looks like the error.message is `'Unhandled Runtime Error\nCache'`